### DISCUSSION TITLE
i meet a question

### DISCUSSION THREAD

**Discussion by Liu0217lu** [pydata/xarray#8822] [Q&A] [answered]
when i use xarry to open a nc file, i meet ValueError: Failed to decode variable 'time': unable to decode time units 'days since 1800-1-1 00:00:00' with 'the default calendar'. Try opening your dataset with decode_times=False or installing cftime if it is not installed.

my code is just xr.opendataset("").the question is yesterday it was ok, i dont change anything ,,today it is not work. I searched some actions to solve it ,such as  decode_times= False, but i need to do groupby('time.dayofyear'),i dont work again.

**Comment by keewis** [comment]
This is not easy to answer because we don't have enough information (and I'm also not an expert on this).

Usually, if something works then stops working there's something that changed, either the data or the software. If you're using `conda`, try `conda list --revision` to see a log of all changes to the environment.

Can post the string representation of `ds["time"]` (`print(ds["time"])`) for `decode_times=False`? That might help figuring out the issue already. Otherwise we need something we can use to reproduce your issue. For example, try loading your data with `decode_times=False` and extract just the time variable, then save that to a file and upload it here.

**Reply by Liu0217lu** [reply, marked as answer]
thanks for your answer. i use decode_times=False and update pandas,i have solved this problem.thank you again
